src/pending_transactions.erl

```diff
@@ -117,12 +117,50 @@ add_transaction(Transaction, PendingTx)
     end.
     
 
-%% @doc Get list of all the players in the container.
+%% doc Get list of all the players in the container.
 
-get_players(_) ->
-    ok.
+%% get_all_players(#pending_tx{player_map = PlMap}) ->
+%%     maps:keys(PlMap).
```

Comment:
just delete it?

Reply:
best way to implement a function is to comment it out!

Reply:
such efficiency